python/tskit/trees.py

```diff
@@ -5522,6 +5522,32 @@ def genetic_relatedness(
         :ref:`polarised <sec_stats_polarisation>`,
         and :ref:`return value <sec_stats_output_format>`.
 
+        What is computed depends on ``mode``:
+
+        "site"
+            Number of pairwise allelic matches between two samples relative to
+            the rest of the samples, per unit of chromosome length. To be
+            precise, let `m(a,b)` denote the total number of pairwise allelic
+            matches between samples `a` and `b`, and let `C` and `D` be
+            independently chosen individuals from the sample. Then, this
+            computes `E[m(a,b) - m(a,C) - m(b,D) + m(C,D)]`. If
+            ``proportion=True``, this corresponds to :math:`K_{c0}` in
+            `Doug & Balding (2014) <https://www.nature.com/articles/nrg3821>`_.
+
+        "branch"
+            Total area of branches ancestral to two samples relative to
+            the rest of the samples. To be precise, let `S(a,b)` denote the
+            shared branch area between samples `a` and `b`, and let `C` and `D`
```

Comment:
maybe "let `S(a,b)` denote the total area of all branches ancestral to both `a` and `b`, ..." since "shared" is ambiguous.